app/javascript/components/recipes/show/base-component.vue

```diff
@@ -96,18 +102,20 @@ export default {
         this.errorResponse(error);
       }
     },
-
     async successResponse(status) {
       this.status = status;
       this.error = '';
     },
-
     async errorResponse(error) {
       this.status = error.response.status;
       this.error = error;
     },
-
   },
 
+  computed: {
+    getPrice() {
+      return this.recipe.ingredients.map(element => this.ingredientFinalPrice(element.quantity, element.price)).reduce((acc, curVal) => acc + curVal, 0);
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>This line has a length of 153. Maximum allowed is 120  vue/max-len